amundsen_common/utils/utils.py

```diff
@@ -0,0 +1,12 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional, TypeVar
```

Comment:
could we move the utils folder to gremlin as well ?

Reply:
Sure!

- [ ] move the utils folder to gremlin